guide/03-the-gis/accessing-and-managing-users.ipynb

```diff
@@ -456,6 +456,9 @@
     "\n",
```

Comment:
<p style="user-select: auto;">I'd prefer if you use the string <code>creatorUT</code>  instead of our internal <code>creator</code>  string</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Hi Atma - I have made the updates as suggested. Please review again.